esmvaltool/recipes/droughts/recipe_spei.yml

```diff
@@ -0,0 +1,89 @@
+
+# ESMValTool
+# recipe_spei.yml
+---
+documentation:
+  title: SPI and SPEI Drought Indices
+
+  description: |
+    Example calculates for the SPI and SPEI drought indices
+
+  authors:
+    - berg_peter
+    - lindenlaub_lukas
+
+  maintainer:
+    - weigel_katja
+
+  projects:
+    - c3s-magic
+
+  references:
+    - acknow_project
+
+CMIP6: &cmip6 {project: CMIP6, mip: Amon, ensemble: r1i1p1f1, grid: gn}
+
+datasets:
+# - {dataset: ERA-Interim, project: OBS6, type: reanaly, version: 1, tier: 3}
+- {<<: *cmip6, dataset: ACCESS-CM2, institute: CSIRO-ARCCSS}
+
+preprocessors:
+  preprocessor:
+    regrid:
+      target_grid: reference_dataset
+      scheme: linear
+  perday:
+    regrid:
+      target_grid: reference_dataset
+      scheme: linear
+    convert_units:
+      units: mm day-1
+
+diagnostics:
+  diagnostic:
+    description: Calculating SPI and SPEI index
+    variables:
+      tasmin: &var
+        reference_dataset: ACCESS-CM2
+        preprocessor: preprocessor
+        start_year: 2000
+        end_year: 2005
+        mip: Amon
+        exp: [historical]
+      tasmax: *var
+      pr:
+        <<: *var
+        preprocessor: perday
+    scripts:
+      spi:
+        script: droughts/spei.R
+        ancestors: [pr]
+        distribution: Gamma
+        smooth_month: 6
+      pet:
+        pet_type: "Hargreaves"  # "Penman_clt"
```

Comment:
I found one more: Remove or update comment "Penman_clt" is outdated, it is "Penman" now